### PR TITLE
Remove all references to print server and monitor server

### PR DESCRIPTION
**Pull request by Si-plus-plus**
Remove all related to:
- `PRINT_SYSTEM_ADDRESS`
- `CUSTOM_PRINT_ENABLED`
- `MONITOR_ADDRESS`
- `MONITOR_URL`
to prevent confusion as they are not being used anymore.

### CODE REVIEW

**Comment by fushar** [comment]
Just realized, can we remove `DRAFT_PRINTER` and `FINAL_PRINTER` as well?

**Review by fushar — CHANGES_REQUESTED** [review]
Just realized, can we remove DRAFT_PRINTER and FINAL_PRINTER as well?

**Review by fushar — CHANGES_REQUESTED** [review]
Also `PRINT_SYSTEM_URL` and `PRINT_SYSTEM_ADDRESS` 🙈

**Review by fushar — APPROVED** [review]

File: Translation/settings.py [hunk 1]
```diff
@@ -212,5 +206,4 @@
 PRINTED_DRAFT_TRANSLATIONS_ROOT = os.path.join(BASE_DIR, 'draft_translations/')
 HOST_URL = 'http://127.0.0.1:9000/'
 
-PRINT_ENABLED = PRINT_SYSTEM_ADDRESS is not None
-CUSTOM_PRINT_ENABLED = PRINT_ENABLED and False
+PRINT_ENABLED = True
```
Comment (raisfathin) [inline Translation/settings.py:209]: Let's remove this too.

**Review by raisfathin — APPROVED** [review]

File: Translation/settings.py (hunk 1 above)
Comment (fushar) [reply Translation/settings.py:209]: Ah... agree!

Comment (Si-plus-plus) [reply Translation/settings.py:209]: Remove the `PRINT_ENABLED` ? But wouldn't it be used later on..?

Comment (fushar) [reply Translation/settings.py:209]: We can assume that it is always true.

Comment (Si-plus-plus) [reply Translation/settings.py:209]: okay

**Pull request merged**